src/arm/BBORG_SERVO-00A2.dts

```diff
@@ -75,45 +75,27 @@
     #address-cells = <1>;
     #size-cells = <0>;
 
-    pca9685_servo: pca9685_serve@7f {
-        compatible = "nxp, pca9685";
+    pca: pca@70 {
+        compatible = "nxp,pca9685-pwm";
+        #pwm-cells = <16>;
+        reg = <0x70>;
         lable = "pca9685_servo";
-        pinctrl-names = "default";
-        pinctrl-0 = <&bborg_servo_pca9685_en_pins>;
-        enable-gpios = <&gpio2 4 GPIO_ACTIVE_HIGH>;
-        reg = <0x7f>;
+        /* invert; */
+        /* open-drain; */
     };
 };
 
 &{/} {
-    gpio_inputs {
-        compatible = "gpio-keys";
+    leds {
```

Comment:
Why is this called leds?

Reply:
I was a different person back then, didn't knew as much as I know now. I will update this accordingly :+1: